### ISSUE TITLE
Releases

### ISSUE THREAD

**Issue by mdaverde** [libbpf/libbpf-rs#281]
Am wondering around the general appetite for more standard release practices. Actions such as:

- Git tags per version
- Github releases
- Changelog generation 

Any of these would reduce friction in determining what changed between crate releases

**Comment by danielocfb** [comment]
I agree that it would be useful to include these bits. Do you have a suggestion on how to go about changelog generation?
